__tests__/integration/send-tx.test.js

```diff
@@ -1359,3 +1359,246 @@ describe('send tx (custom tokens)', () => {
     done();
   });
 });
+
+describe('filter query + custom tokens', () => {
+  let wallet1;
+  const bugCoin = {
+    name: 'BugCoin',
+    symbol: 'BUG',
+    uid: null
+  };
+
+  beforeAll(async () => {
+    wallet1 = new WalletHelper('filter-custom-1');
+
+    await WalletHelper.startMultipleWalletsForTest([wallet1]);
+
+    // Inject 10 HTR into wallet1 and invest them all creating the custom token
+    await wallet1.injectFunds(10, 0);
+    const createTokenTx = await wallet1.createToken({
+      name: bugCoin.name,
+      symbol: bugCoin.symbol,
+      amount: 1000,
+      address: await wallet1.getAddressAt(0),
+    });
+    bugCoin.uid = createTokenTx.hash;
+
+    // Inject 20 HTR on other address to validate that it's not changed by custom token txs
+    await wallet1.injectFunds(20, 1);
+
+    /*
+     * Status:
+     * addr0: 0 HTR, 1000 BUG
+     * addr1: 20 HTR
+     */
+  });
+
+  afterAll(async () => {
+    await wallet1.stop();
+  });
+
+  it('should reject for insufficient funds (custom)', async done => {
+    // Sending all the tokens to facilitate address-info validation
+    const txErr = await wallet1.sendTx({
+      inputs: [{ type: 'query', filter_address: await wallet1.getAddressAt(0) }],
```

Comment:
I would just add a new error test case with amount 100 (an amount that the wallet has) but with an address without the utxo, e.g. the address 1.